boards/openmote-b/Makefile.include

```diff
@@ -9,27 +9,32 @@ PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbserial*)))
 # Set default flash tool
 export PROGRAMMER ?= cc2538-bsl
 
+FLASHFILE ?= $(HEXFILE)
+RESET_FLAGS ?= $(BINDIR)
+export OBJDUMPFLAGS += --disassemble --source --disassembler-options=force-thumb
+
 ifeq ($(PROGRAMMER),cc2538-bsl)
   OS := $(shell uname)
   ifeq ($(OS),Linux)
     PORT_BSL ?= $(PORT_LINUX)
   else ifeq ($(OS),Darwin)
     PORT_BSL ?= $(PORT_DARWIN)
   endif
-  FLASHER = $(RIOTBASE)/dist/tools/cc2538-bsl/cc2538-bsl.py
-  FFLAGS  = -p "$(PORT_BSL)" --bootloader-invert-lines -e -w -v -b 460800 $(FLASHFILE)
+  FLASHER ?= $(RIOTBASE)/dist/tools/cc2538-bsl/cc2538-bsl.py
```

Comment:
Changing all the variables to `?=` is unrelated to the current fix

Reply:
I just think it is silly to not allow override.  Should I remove?

Reply:
Either there is a testing procedure for this and each changed commands here or it should be a different PR. Otherwise it is untested.

Also many other boards should also be updated and would benefit from a batch work.